core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/infra/trackranges/RJSDirectionalTrackRange.java

```diff
@@ -0,0 +1,17 @@
+package fr.sncf.osrd.railjson.schema.infra.trackranges;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import fr.sncf.osrd.railjson.schema.common.graph.EdgeDirection;
+
+@SuppressFBWarnings({"UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD"})
+public class RJSDirectionalTrackRange extends RJSTrackRange {
+    public EdgeDirection direction;
+
+    /**
+     * Constructor
+     */
```

Comment:
This is useless

Reply:
The style checker doesn't accept undocumented public methods, and there's not a lot of details to add here. It's not ideal but not useless

Reply:
ok I got it :)